### PR TITLE
[WIP] Update RHPD Demo to oadp-1.1

### PR DESCRIPTION
**Pull request by hhpatel14**
##### SUMMARY

Update OADP from v0.5.1 to v1.1.3

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

role: ocp4-workload-oadp

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: ansible/roles/ocp4-workload-oadp/defaults/main.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 # workload vars
-oadp_operator_subscription_release: v0.5.1
+oadp_operator_subscription_release: v1.1.3
 oadp_operator_subscription_startingcsv: oadp-operator.{{ oadp_operator_subscription_release }}
-oadp_operator_subscription_source: community-operators
-oadp_operator_subscription_sourcenamespace: openshift-marketplace
-oadp_operator_subscription_name: oadp-operator
-oadp_operator_subscription_channel: stable
+oadp_operator_subscription_source: redhat-operators
+oadp_operator_subscription_sourcenamespace: openshift-adp
```
Comment (kaovilai) [inline ansible/roles/ocp4-workload-oadp/defaults/main.yml:6]: Based on this I think namespace here is wrong
```yaml
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  annotations:
    operatorframework.io/managed-by: marketplace-operator
    target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
  name: redhat-operators
  namespace: openshift-marketplace
```

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: ansible/roles/ocp4-workload-oadp/tasks/post_workload.yml [hunk 2]
```diff
@@ -61,6 +61,7 @@
         apps_url: "{{ apps_url | default('appsurl') }}"
       # yamllint disable-line rule:line-length
         cluster_admin_user: "{{ cluster_admin_user | default(ocp4_workload_oadp_cluster_admin) | default('kubeadmin') }}"
+        cluster_admin_password: "{{ cluster_admin_password | default(ocp4_workload_oadp_cluster_admin_password) | default('undefined') }}"
         api_url: "{{ api_url_results.stdout }}"
         ocp4_password: "{{ student_password | default('undefined')}}"
```
Comment (kaovilai) [inline ansible/roles/ocp4-workload-oadp/tasks/post_workload.yml:66]:
```suggestion
        cluster_admin_password: "{{ cluster_admin_password | default(ocp4_workload_oadp_cluster_admin_password) | default('undefined') }}"
        api_url: "{{ api_url_results.stdout }}"
```

if you prefer to keep the var names accurate to agnosticd in bookbag then perhaps just remove ocp4_password here entirely. this means you must change the var in the bookbag also.. alternatively stuff cluster_admin_password back into ocp4_password.

Comment (hhpatel14) [reply ansible/roles/ocp4-workload-oadp/tasks/post_workload.yml:66]: I think cluster-admin_password is less confusing than ocp4_password. Also, MTC is using ocp4_password = student password, so let's add new variable instead changing ocp4_password. We might need 'student password' in future.

**Review by kaovilai — APPROVED** [review]

File: ansible/roles/ocp4-workload-oadp/tasks/post_workload.yml (hunk 2 above)
Comment (kaovilai) [reply ansible/roles/ocp4-workload-oadp/tasks/post_workload.yml:66]: @hhpatel14 click resolve.

**Pull request closed without merging**